presto-orc/src/main/java/com/facebook/presto/orc/reader/StreamReader.java

```diff
@@ -30,7 +30,8 @@
     enclosing query. Fields not mentioned should be returned as null
     RLEs so as to preserve record layout without materializing the
     data. 'depth' is the position corresponding to this StreamReader
-    in subfields.get(n).getPath(). If subfields is {"a.b.c", "a.b.d"} and this is the StrucStreamReader of b then depth is 1.
+    in subfields.get(n).getPath(). If subfields is {"a.b.c", "a.b.d"}
+    and this is the StrucStreamReader of b then depth is 1.
```

Comment:
`StructStreamReader`

Reply:
@tdcmeehan I'm seeing multiple readers using this API, e.g. List, Map, MapDirect and StructStreamReader.

Reply:
I was just pointing out a typo in the comment